.github/workflows/ci.yml

```diff
@@ -114,7 +114,9 @@ jobs:
       if: matrix.os == 'ubuntu-latest'
 
     - name: yarn unit
-      run: xvfb-run --auto-servernum yarn unit:cicoverage
+      run: |
+        xvfb-run --auto-servernum yarn unit:cicoverage
+        yarn c8 report --reporter text-lcov > unit-coverage.lcov
```

Comment:
unit coverage didn't matter for this since we don't pass extra flags to it, but it made sense to do both in the same way.

Reply:
much better than doing in package.json!